RLTest/env.py

```diff
@@ -184,6 +185,8 @@ def getEnvByName(self):
         }
 
         if self.env == 'oss':
+            if Defaults.randomize_ports:
+                kwargs['port'] = 0
```

Comment:
Should also be at the enterprise env here bellow ...

Reply:
Do we need more ports in the enterprise?

Reply:
its enterprise not enterprise cluster, should be the same as oss